client/gutenberg/extensions/presets/jetpack/package.json

```diff
@@ -1,8 +1,11 @@
 {
 	"name": "@automattic/jetpack-blocks",
-	"version": "0.1.0",
+	"version": "1.0.0-alpha.0",
 	"description": "Gutenberg blocks for the Jetpack WordPress plugin",
-	"main": "editor.js",
+	"main": "build/editor.js",
+	"files": [
+		"build"
```

Comment:
Super-nitpicky: Can we append a slash to make clear it's a dir? (Might be useful if we start adding singular files as well)